src/sentry/monitors/serializers.py

```diff
@@ -6,22 +6,68 @@
 from sentry.api.serializers import ProjectSerializerResponse, Serializer, register, serialize
 from sentry.models import Project
 
-from .models import Monitor, MonitorCheckIn
+from .models import Monitor, MonitorCheckIn, MonitorEnvironment
+
+
+@register(MonitorEnvironment)
+class MonitorEnvironmentSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        return {
+            "monitor_id": obj.monitor.id,
```

Comment:
do I also need to run a `select_related` for `monitor`? I hate the idea of pulling the same object out of the db twice but i feel like this is doing that

Reply:
You can do `obj.monitor_id` here and avoid the db call